testutil/yaml/yaml.go

```diff
@@ -0,0 +1,31 @@
+package yaml
```

Comment:
Are we not already adding this in another PR?

Reply:
Yes, It's the same as in other PRs and is needed to have a successful build.

It will end-up being the same file when all the PRs get merged.